docs/user/registering-microshift-devices-acm.md

```diff
@@ -0,0 +1,157 @@
+# Auto-Registering Devices with MicroShift into ACM
+
+## Configuring RHACM Agent Registration
+
+Auto-registration of devices with MicroShift into ACM relies on a feature called agent registration, allowing clusters to be imported via REST API calls using a CA bundle and token. Follow these [instructions](https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.11/html/clusters/cluster_mce_overview#importing-managed-agent) to configure the necessary RBAC policies for your own user. Skip step 7 of the instructions in section 1.6.5.2.3, as this step will be automatically handled by the fleet devices.
```

Comment:
Section 1.6.5.**3**.3. Referencing a section (or even step) by number will frequently break, though. Can we just describe what needs to be skipped ("Skip the step to import the managed cluster to the hub cluster").

Reply:
+1